Packs/Salesforce/Scripts/SalesforceCaseStatusPicklist/SalesforceCaseStatusPicklist.py

```diff
@@ -0,0 +1,6 @@
+import demistomock as demisto  # noqa: F401
+from CommonServerPython import *  # noqa: F401
+
+picklistValues = demisto.executeCommand("salesforce-describe-sobject-field",
+                                        {"sobject": "Case", "field": "Status"})[0]['Contents'].get('picklistValues')
+demisto.results({'hidden': False, 'options': [value.get('label') for value in picklistValues]})
```

Comment:
use CommandResults here please

Reply:
I guess you want to say return_results instead of CommandResults

Reply:
This comment is not relevant anymore because the Pack's support level has changed to - 'community'.